types/openlayers/index.d.ts

```diff
@@ -7132,6 +7132,12 @@ declare module ol {
         pointerEvent: ol.pointer.PointerEvent;
     }
 
+    interface DrawEvent {
```

Comment:
Sorry, this is the link http://openlayers.org/en/master/apidoc/ol.interaction.Draw.Event.html